docs/resources/redis_cluster.md

```diff
@@ -87,6 +111,24 @@ The `acl` block supports:
 
 - `settings` - (Optional) Map of settings for redis cluster. Available settings can be found by listing redis versions with scaleway API or CLI
 
+
+- `private_network` - (Optional) Describes the private network you want to connect to your cluster. If not set, a public network will be provided.
+
+The `private_network` block supports :
+- `id` - (Required) The UUID of the private network resource.
+- `service_ips` - (Required) Endpoint IPv4 addresses in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation). You must provide at least one IP per node.
+
+~> **Important:** The way to use private networks differs whether you are using redis in standalone or cluster mode.
+- Standalone mode (`cluster_size` = 1) : you can attach as many private networks as you want (each must be a separate block). If you detach your only private network, your cluster won't be reachable until you define a new private or public network. You can modify your private_network and its specs, you can have both a private and public network side by side.
+- Cluster mode (`cluster_size` > 1) : you can define a single private network as you create your cluster, you won't be able to edit or detach it afterwards, unless you create another cluster. Your `service_ips` must be listed as follows:
+```hcl
+  service_ips = [
```

Comment:
Should we use addresses like: 
"10.12.1.10/24",
"10.12.1.11/24",
"10.12.1.12/24",

like @abarbare mentionned?